homeassistant/auth/models.py

```diff
@@ -106,6 +106,8 @@ class RefreshToken:
     last_used_at: Optional[datetime] = attr.ib(default=None)
     last_used_ip: Optional[str] = attr.ib(default=None)
 
+    cred: Optional["Credentials"] = attr.ib(default=None)
```

Comment:
Let's not abbreviate so it's clear what it contains. 
```suggestion
    credentials: Optional["Credentials"] = attr.ib(default=None)
```

Reply:
just really annoying with the "s" at the end. Since we also have arrays of credential(s)s. No singular of the full name. Will do my best.